src/pages/proposals/new-cards.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Pagination } from '@material-ui/lab';
+import styled from '@emotion/styled';
+import { RootState } from 'src/reducers/store';
+import { useHistory } from 'react-router-dom';
+import { voteCalculation } from 'src/utils/vote-calculation';
+import TooltipGraphSection from './tooltip-graph-section';
+import { statusColor } from 'src/utils/color-formatter';
+import moment from 'moment';
+
+type CardsProps = {
+	home?: boolean;
+	proposals: any[];
+};
+
+const NewCards: FunctionComponent<CardsProps> = ({ home, proposals }) => {
+	const history = useHistory();
+	const [page, setPage] = useState(1);
+	let rowsPerPage = 15;
+	if (home) {
```

Comment:
You can delete `home` probably, doesn't seem like its used

Reply:
should lint fix that or spot it? @Ericky14